### DISCUSSION TITLE
customize the signaling server in RustDesk on our Linux machine.

### DISCUSSION THREAD

**Discussion by linuxserver2016** [rustdesk/rustdesk-server-pro#400] [Q&A]
- I have cloned Rustdesk to work properly in linux. But I need to customize the signaling server in RustDesk on our Linux machine.
- The signaling server is typically included by default or not included.
- If available How to customize the signaling server

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/
